cpu/efm32/periph/timer_series01.c

```diff
@@ -47,6 +50,73 @@
  */
 static timer_isr_ctx_t isr_ctx[TIMER_NUMOF];
 
+/* Bitmask storing which channel are periodic and which not. Only use
+ * getter/setter functions to access */
+static uint8_t is_periodic_mask[TIMER_NUMOF];
+/* Variable holding if the top value of the timer has been modified and by
+ * which. Only use getter/setter functions to access */
+static uint8_t modified_top_mask[TIMER_NUMOF];
+
+/* Check if the given channel on the given timer is periodic */
+static bool is_periodic(tim_t dev, int chan)
+{
+    assume((unsigned)chan < 8);
```

Comment:
```suggestion
    assume((unsigned)chan < CHANNEL_NUMOF);
```

Reply:
I really should have added a comment here. The current implementation assumes that one byte per timer is enough as bitmask for all channels of that timer. That assumption may not hold on future silicon and this is intended to catch that.

I'll add the missing comment.